server/tokenapi/DBUtils.go

```diff
@@ -0,0 +1,18 @@
+package tokenapi
+
+import (
+	"github.com/akyoto/uuid"
+	"github.com/animenotifier/notify.moe/arn"
+)
+
+func GetUserFromToken(token uuid.UUID) *arn.User {
+	user := &arn.User{}
+
+	for user = range arn.StreamUsers() {
```

Comment:
Could replace this O(n) loop with an O(1) lookup using a new collection in the DB ("TokenToUser") later on.

Otherwise this can become a huge bottleneck.

Reply:
I originally wanted to implement this in a better way but I'm honestly too unfamiliar with nano so I didn't want to touch what I was unsure about.